playground/bicep/BicepSample.AppHost/Program.cs

```diff
@@ -0,0 +1,65 @@
+using Aspire.Hosting.Azure;
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddAzureProvisioning();
+
+var parameter = builder.AddParameter("val");
+
+var templ = builder.AddBicepTemplate("test", "test.bicep")
+                   .WithParameter("test", parameter)
+                   .WithParameter("values", ["one", "two"]);
+
+var kv = builder.AddBicepKeyVault("kv");
```

Comment:
For this Azure specific resource types is the plan to completely replace the existing keyvault resource types (and rename these new ones). The fact that they are BicepResource types shouldn't be a factor right?

Reply:
Yep